Tests/docker_native_image_config.json

```diff
@@ -99,13 +134,14 @@
          "id":"Rasterize",
          "reason":"Issue: CRTX-72338.",
          "ignored_native_images":[
+            "native:8.2",
             "native:8.1"
          ]
       }
    ],
    "flags_versions_mapping": {
       "native:dev" : "native:dev",
-      "native:ga" : "native:8.1",
-      "native:maintenance" : ""
+      "native:ga" : "native:8.2",
+      "native:maintenance" : "native:8.1"
```

Comment:
approved, but lets double check with @sslellouche that 8.1 actually still needs support. From my understanding in 8.1 the feature is completely disabled